editions/tw5.com/tiddlers/concepts/Story Tiddler Template.tid

```diff
@@ -6,6 +6,8 @@ type: text/vnd.tiddlywiki
 
 "Story tiddler template" refers to the template used to display a tiddler within the story river.
 
+This is the core story tiddler template: $:/core/ui/StoryTiddlerTemplate
```

Comment:
It would clearer to say "The core story tiddler template can be found in $:/core/ui/StoryTiddlerTemplate"

Reply:
I don't understand how to modify this PR or to retract/cancel it. I've posted a new one for the same tiddler. Please close/delete/dismiss/whatever this one. Thanks.